src/sentry/integrations/opsgenie/integration.py

```diff
@@ -64,24 +63,29 @@
 class InstallationForm(forms.Form):
     base_url = forms.ChoiceField(
         label=_("Base URL"),
-        # help_text=_("Either https://api.opsgenie.com/ or https://api.eu.opsgenie.com/"),
-        # widget=forms.TextInput(attrs={"placeholder": "https://api.opsgenie.com/"}),
         choices=[
             ("https://api.opsgenie.com/", "api.opsgenie.com"),
             ("https://api.eu.opsgenie.com/", "api.eu.opsgenie.com"),
         ],
     )
+    provider = forms.CharField(
+        label=_("Account Name"),
+        help_text=_("Example: 'hello' for https://hello.app.opsgenie.com/"),
+        widget=forms.TextInput(),
+    )
+
     api_key = forms.CharField(
-        label=("Opsgenie API Key"),
+        label=("Opsgenie Integration Key"),
+        help_text=_(
+            "Optional: add an integration key for sending alerts. You can rename this key later."
```

Comment:
Since the field doesn't say `[REQUIRED]` in the UI, you don't have to add the `Optional:` call out in this format. You could change the text to 'Optionally add ...' to make it more conversational.

Reply:
I'd also change `add an integration key for...` to `add your first integration key for...` to put more emphasis into the fact that it's not a specific key, just an starting one.